src/rpc/rpccategory.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <rpc/rpccategory.h>
+
+namespace rpccategory
+{
+
+std::string Label (const RPCCategory category);
```

Comment:
Remove this line

Reply:
do you mean the empty line (line 10) between `{` and `std::string`?